perun-base/src/main/resources/perun-roles.yml

```diff
@@ -2198,55 +2198,76 @@ perun_policies:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Group
+      - MFA: Member
 
   deleteMembers_List<Member>_policy:
     policy_roles:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Group
+      - MFA: Member
 
   deleteAllMembers_Vo_policy:
     policy_roles:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Group
+      - MFA: Member
 
   createSpecificMember_Vo_Candidate_List<User>_SpecificUserType_List<Group>_policy:
     policy_roles:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA:
```

Comment:
discussed with @balcirakpeter and checking if VO is critical is enough, no need for global MFA check. please check also other methods where you check mfa globaly if it makes sense to check just vo. also, besides checking if member or user is critical in the other methods, vo should be checked too, so please add it where it makes sense.

Reply:
when i think about it again, i'd add also Group in the createSpecificMember and createMember rules

Reply:
Should the Group be added to the create Sponsor Member methods?

Reply:
yes, if they are assigned to groups

Reply:
I think we should remove `- MFA: Group` from `createSponsoredMember_Vo_String_Map<String_String>_String_User_LocalDate_policy`, `createSponsoredMembersFromCSV_Vo_String_List<String>_User_policy`, `createSponsoredMembers_Vo_String_List<String>_User_policy` because it shouldn't affect groups anyhow. Besides that the rest looks all good to me